charts/fluent-bit/values.yaml

```diff
@@ -44,13 +44,15 @@ podSecurityPolicy:
   create: false
   annotations: {}
 
-openShift:
-  # Sets Openshift support
+openshift:
```

Comment:
It's coming from personal experience: internal OpenShift applications and services use lower-case at most (caused by k8s restrictions, application naming restrictions). From operational perspective it could looks weird.

Reply:
But this is the Helm config and Helm YAML idiomatically uses camel case; e.g. `cert-manager` would be represented by the key `certManager`. Also this is already used and I see no reason to change it.

Reply:
Ok, will keep as-is